src/OrchardCore.Themes/TheAdmin/Assets/scss/bootstrap-switch.scss

```diff
@@ -174,3 +174,13 @@ $switch-transition: .2s all !default;
         margin-left: 1rem;
     }
 }
+.switch
```

Comment:
I don't think we need this selector. It can be applied everywhere we have a hint.

Reply:
@Skrypt - hints on TextField and other such fields don't have – at all which is why I was going through being more specific with the rule. Unless you just want me to add a modifier like .hint--dash and just add the modifier to the relevant templates and then do:

`.hint--dash::before { content: "–" }`

Reply:
At some point we included the "double dash" in the translation files and we started removing them because it was not necessary to be translated. I think we should have a dash everywhere we use a .hint unless you say that it would look awkward in some place.

Reply:
No one has taken this task so far because you will also need to remove these in the localization.po files.

Reply:
How about I try it just on .hint and see what it looks like. By the way, I can't see a gulp, grunt or npm command in TheAdmin theme...  how is the scss compiled?

Reply:
It's built from the main `gulp build` root command. If it doesn't compile, you need to make a change in the TheAdmin.scss file for the gulp task to pick it up "small issue I know".

Reply:
The hint is used in a lot of places. Adding it simply to .hint adds it into the media picker and everything:

![screencapture-localhost-5001-Admin-Contents-ContentTypes-Article-Create-2020-10-21-17_12_50](https://user-images.githubusercontent.com/2867511/96685641-bec28600-13c0-11eb-8684-29f99ce71958.png)

Reply:
And even here... personally I think "Last modified 2 minutes ago by admin" should actually have it's own class rather than re-using hint even if it a hint reused the same text styles.

<img width="610" alt="Screen Shot 2020-10-21 at 5 24 59 pm" src="https://user-images.githubusercontent.com/2867511/96686832-64c2c000-13c2-11eb-83eb-32f476117982.png">

Reply:
Ok then we need a class to specify which one should be dashed and we need to redo them one by one.
`<span class="hint dashed">hello</span>`
`.hint.dash::before { content: "—" }`

Reply:
Oh boy.

Reply:
🙈 🙉 🙊 Time to go sleep for me 😉